src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -125,10 +130,61 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
+    @GET
+    @Path("{instanceName}/admin")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAdminList(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        List<String> instanceGithubTeams = instance.getSpec().getGithub().getTeams();
+
+        List<Object> instanceTeamMembers = new ArrayList<>();
+        List<User> allInstanceAdmins = new ArrayList<>();
+
+        TeamService teamService = new TeamService(client);
+
+        // Loop though all teams in the Kabanero CRD orginization and check to if the team exists in the list of Kabanero crd teams
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            for (String instanceAdminTeam : instanceGithubTeams) {
+                if (instanceAdminTeam.equals(orgTeam.getName())) {
+                    // If the team exits in the CRD create an object with the team name, id and a
+                    // list of memebrs that belong to the team
+                    JsonObject teamObject = new JsonObject();
+                    teamObject.addProperty("name", orgTeam.getName());
+                    teamObject.addProperty("id", orgTeam.getId());
+                    teamObject.add("members", new Gson().toJsonTree(teamService.getMembers(orgTeam.getId()), new TypeToken<List<User>>() {}.getType()));
+                    instanceTeamMembers.add(teamObject);
+                    // Loop through each team and get all members that belong to that team make sure
+                    // they dont alreay exist in the list of all admins and add the user.
+                    for (User member : teamService.getMembers(orgTeam.getId())) {
+                        if (!allInstanceAdmins.stream().anyMatch(admin -> member.getLogin().equals(admin.getLogin()))) {
+                            allInstanceAdmins.add(member);
+                        }
+                    }
```

Comment:
so this I think we should move to the front end, less processing on the backend is best - since the backend really shouldn't care too much about formatting, just getting the correct data.

Reply:
so renaming `instanceTeamMembers` -> `adminMembers` and returning `adminMembers` should be good